### PR TITLE
Replace (some) of First & Last Name variable example references (Batch 2)

### PR DESCRIPTION
**Pull request by hergaiety**
## Issue
https://github.com/ember-learn/guides-source/issues/1480
Prior batch: https://github.com/ember-learn/guides-source/pull/1530

TL;DR Replacing `firstName` and `lastName` example code with `title` and `name`, or just simply `name`, or something else entirely for i18n/W3C/respect concerns.

## Pages touched in this batch of changes
* [x] [Configuration - Disabling Prototype Extensions - Functions](https://guides.emberjs.com/release/configuring-ember/disabling-prototype-extensions/#toc_functions)
* [x] [Getting Started - Working with HTML, CSS, and JavaScript - JavaScript Classes](https://guides.emberjs.com/release/getting-started/working-with-html-css-and-javascript/#toc_javascript-classes)
* [x] [Upgrading - Octane Upgrade Guide - Tracked Properties](https://guides.emberjs.com/release/upgrading/current-edition/tracked-properties/)
* [x] [Application Concerns - Run Loop - Why is the run loop useful?](https://guides.emberjs.com/release/applications/run-loop/#toc_why-is-the-run-loop-useful)
* [x] [In-Depth Topics - Autotracking In-Depth - Tracked Properties in Custom Classes](https://guides.emberjs.com/release/in-depth-topics/autotracking-in-depth/#toc_tracked-properties-in-custom-classes)

## Notes

I don't feel this needs to go back prior to `/release` but I'm open to a suggested amount of backporting.

Open to suggested alternatives for these changes as well.

### CODE REVIEW

File: guides/release/upgrading/current-edition/tracked-properties.md [hunk 1]
```diff
@@ -228,34 +228,34 @@ Likewise, properties that are not decorated with `@tracked` that you get using
...
+    return this.width / this.height;
   }
 }
 
-let kris = new Person();
-set(kris, 'firstName', 'Kris');
-set(kris, 'lastName', 'Selden');
+let profilePhoto = new Image();
+set(profilePhoto, 'width', 259);
+set(profilePhoto, 'height', 259);
```
Comment (ijlee2) [inline guides/release/upgrading/current-edition/tracked-properties.md:258]: I think you might have meant the number to end in a 0.
```suggestion
set(profilePhoto, 'width', 250);
set(profilePhoto, 'height', 250);
```

**Review by ijlee2 — APPROVED** [review]
I skimmed through the changes by looking for patterns. I'm not sure if some of the examples (e.g. `Image.create()`) will still work in Octane, but think investigating this is outside of the ticket scope.
